tests/functional/test_article.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from pages.article import ArticlePage
+
+
+# page title
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_title_in_title(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.title_in_title(selenium)
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_title_text_has_length(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.title_text_has_length(selenium)
+
+
+# article
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_article_is_displayed(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.article_is_displayed
+
+
+# page buttons
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_language_menu_is_displayed(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.language_menu_is_displayed
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_edit_button_is_displayed(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.edit_button_is_displayed
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_advanced_menu_is_displayed(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.advanced_menu_is_displayed
+
+
+"""
+The tests below are duplicates of ones in test_home, how can I avoid this?
```

Comment:
> Some duplication in tests is OK. @jgmize says it is a good thing.

Jay Fields explains better than I could: http://blog.jayfields.com/2008/05/testing-duplicate-code-in-your-tests.html

> [...] This can potentially save more lines, but would involve rewriting test functions as classes and may confuse things. There's also not a lot of +1 for that blog post, which may mean it isn't really the py.test philosophy.

I'm -1 on that approach.